pkg/pillar/cmd/volumemgr/blob.go

```diff
@@ -536,6 +536,16 @@ func populateInitBlobStatus(ctx *volumemgrContext) {
 			log.Functionf("populateInitBlobStatus: blob %s in CAS could not get mediaType", blobInfo.Digest)
 			continue
 		}
+		// In Kubevirt eve, we share same user containerd repository between eve and k3s containers.
+		// So volumemgr should ignore publishing blobs which are not downloaded through eve download mechanism
+		// In /run/volumemgr/BlobStatus we should only see the blobs downloaded by eve.
+		if ctx.hvTypeKube {
+			label := blobInfo.Labels
```

Comment:
Why do we need to track the labels in the blobInfo? That breaks modularity = information hiding.